rotkehlchen/exchanges/okx.py

```diff
@@ -51,6 +52,15 @@
 log = RotkehlchenLogsAdapter(logger)
 
 
+class OkxEndpoint(Enum):
+    CURRENCIES = 'currencies'
```

Comment:
I meant to do this

```suggestion
    CURRENCIES = '/api/v5/asset/currencies'
```

since afaik you etierh compare to the key or use the value

Reply:
ah, yes this simplifies it.